runtime/parachains/src/disputes.rs

```diff
@@ -530,11 +530,7 @@ impl DisputeStateFlags {
 		let supermajority_threshold = supermajority_threshold(n);
 
 		let mut flags = DisputeStateFlags::default();
-		let all_participants = {
-			let mut a = state.validators_for.clone();
-			*a |= state.validators_against.iter().by_val();
-			a
-		};
+		let all_participants = state.validators_for.clone() | state.validators_against.clone();
```

Comment:
`|=` is not available any more, but I believe the replacement I wrote here achieves the same result.

Reply:
https://docs.rs/bitvec/1.0.0/bitvec/vec/struct.BitVec.html#impl-BitOrAssign%3C%26%27_%20BitVec%3CT%2C%20O%3E%3E

Please keep the code as close as possible to the previous version. Refactoring can be done in a separate PR.
I don't know for sure what's causing the failing dispute test, but we need to exclude this factor.

Reply:
`|=` is not supported for `.iter().by_vals()` anymore. `|=` is implemented for BitSlice and BitVec rhs, later borrowed as a BitSlice. `|` is implemented using `|=`, it is just more verbose using it. Also, all the unit tests testing this and the following change are passing and cover these pieces of code quite well.

```
test disputes::tests::test_contains_duplicates_in_sorted_iter ... ok
test disputes::tests::test_has_supermajority_against ... ok
test disputes::tests::test_decrement_spam ... ok
test disputes::tests::test_dispute_state_flag_from_state ... ok
test disputes::tests::assure_no_duplicate_statements_sets_are_fine ... ok
test disputes::tests::test_import_new_participant_spam_inc ... ok
test disputes::tests::test_import_slash_against ... ok
test disputes::tests::test_import_prev_participant_spam_dec_confirmed_slash_for ... ok
test disputes::tests::test_check_signature ... ok
```

I am kind of puzzled what causes breakage in the dispute integration tests, but it seems to be more subtle than these changes themselves.